ingest/settings.py

```diff
@@ -0,0 +1,64 @@
+# File is responsible for defining globals and initializing them
+from mongo_connection import MongoConnection
+from bson.objectid import ObjectId
+
+MONGO_CONNECTION = MongoConnection()
+
+
+def init(study_id, study_file_id):
+    global study
+    study = Study(study_id)
+    global study_file
+    study_file = StudyFile(study_file_id)
+
+
+def get_study():
+    global study
+    return study
+
+
+def get_study_file():
+    global study_file
+    return study_file
+
+
+class Study:
+    STUDY_ACCESSION: str = None
+
+    def __init__(self, study_id):
+        Study.update_study_accession(study_id)
+
+    @classmethod
+    def update_file_type(cls, file_type):
+        cls.FILE_TYPE = file_type
+
+    @classmethod
+    def update_study_accession(cls, study_id):
+        study_accession = list(
+            MONGO_CONNECTION._client["study_accessions"].find(
+                {"study_id": ObjectId(study_id)}, {"accession": 1, "_id": 0}
+            )
+        ).pop()
+        cls.STUDY_ACCESSION = study_accession["accession"]
+
+
+class StudyFile:
+    FILE_TYPE: str = None
+    FILE_SIZE: int = None
+    STUDY_FILE_ID = None
+    FILE_NAME = None
+
+    def __init__(self, study_file_id):
```

Comment:
similar to above, 'fetch_study' is probably a more descriptive name for .update.  Then it can just store the document in a variable STUDY_FILE, and you can make getters for the things you need

Reply:
See [comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/163#discussion_r522325975)